pyramda/iterable/reject.py

```diff
@@ -0,0 +1,14 @@
+from pyramda.function.curry import curry
+from . import filter
+
+
+@curry
+def reject(p, xs):
+    """
+    Acts as a complement  of `filter`
+
+    :param p: predicate
+    :param xs: Iterable. A sequence, a container which supports iteration or an iterator
+    :return: list
+    """
+    return list(set(xs) - set(filter(p, xs)))
```

Comment:
I think using sets here means the filter won't respect the order of the input list and will remove duplicates. It would likely be easier to just define `reject` in terms of `filter` with a negated predicate. There should be a test for duplicate elements to ensure it works.